locale/zh/solar-productivity.cfg

```diff
@@ -26,3 +26,10 @@ solar-productivity-1=太阳能电池板和蓄电池的效率： [font=default-se
 solar-productivity-2=太阳能电池板和蓄电池的效率： [font=default-semibold][color=green]+10%[/color][/font].
 solar-productivity-3=太阳能电池板和蓄电池的效率： [font=default-semibold][color=green]+10%[/color][/font].
 solar-productivity-4=太阳能电池板和蓄电池的效率： [font=default-semibold][color=green]+5%[/color][/font].
+
+[mod-setting-name]
+sp:interval=[font=default-semibold][color=blue]更新间隔[/color][/font] (s)
```

Comment:
please change it to `sp-interval` to follow the latest name pattern, `:` is not allow in 2.0 anymore, that would prevent the game from starting